### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by RikusW**

### CODE REVIEW

**Comment by andrew-elder** [comment]
Reminder that all pull requests got to/through open-avb-next. Please resubmit this one. Periodically open-avb-next is merged in to master. The idea is that any issues that might arise are corrected in the open-avb-next branch and that the master branch remains stable. Thanks.

File: .gitignore [hunk 1]
```diff
@@ -48,4 +48,11 @@ bin
 *.orig
 *.o
 
+*.cmake
+*Makefile
+*CMakeFiles/
+CMakeCache.txt
+controller/app/cmdline/avdecccmdline
+
+
```
Comment (dbrignoli) [inline .gitignore]: Hi @RikusW, when re-submitting please remove these 2 extra blank lines. Thanks

**Comment by RikusW** [comment]
I don't see any obvious ways of getting this into open-avb-next....

**Comment by andrew-elder** [comment]
Duh - my bad. Head in the wrong place.... I meant "staging"...

**Pull request closed without merging**